### PR TITLE
Add Leap Ports

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dvaleev@suse.com>

### CODE REVIEW

File: commit.sh [hunk 1]
```diff
@@ -10,6 +10,9 @@ case $proj in
 	Leap:42.1) arches="x86_64"
             repo="standard"
                 ;;
+	Leap:42.1:Ports) arches="ppc64le"
```
Comment (dirkmueller) [inline commit.sh]: can you include aarch64 here?

File: doit.sh [hunk 2]
```diff
@@ -16,6 +16,9 @@ case $proj in
         Leap:42.1) arches="x86_64"
                 repo="standard"
                 ;;
+        Leap:42.1:Ports) arches="ppc64le"
```
Comment (dirkmueller) [inline doit.sh]: can you include aarch64 here?

**Comment by k0da** [comment]
updated with aarch64

**Comment by dirkmueller** [comment]
@k0da can you add 42.2:Ports ?

**Comment by k0da** [comment]
@dirkmueller done

**Comment by dirkmueller** [comment]
LGTM, :+1: 

@aplanas @lnussel can you review  + merge please?

**Pull request merged**